tutorial.md

```diff
@@ -148,9 +148,7 @@ Be sure to add the new resources to your `flutter.yaml` file.
 
```

Comment:
The line before should also be updated, otherwise it's confusing. But actually, I think `flutter create` sets this already, so we might be able to remove this whole part, or at least just replace it with mentioning the role that `uses-material-design` has.